controllers/configmap/configmap_controller.go

```diff
@@ -0,0 +1,154 @@
+package configmap
+
+import (
+	"context"
+
+	kedav1alpha1 "github.com/kedacore/keda-olm-operator/api/v1alpha1"
+	"github.com/kedacore/keda-olm-operator/controllers/util"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/controller-runtime/pkg/source"
+)
+
+const (
+	configMapName      = "keda-metrics-apiserver"
+	configMapNamespace = "keda"
+)
+
+var log = logf.Log.WithName("controller_configmap")
+
+// Add creates a new ConfigMap Controller and adds it to the Manager. The Manager will set fields on the Controller
+// and Start it when the Manager is Started.
+func Add(mgr manager.Manager) error {
```

Comment:
This should be migrated to the same layout as KedaController is

Reply:
ie. https://github.com/kedacore/keda-olm-operator/blob/master/controllers/kedacontroller_controller.go#L171